src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -16,36 +16,42 @@ redirects:
   - /docs/alerts-applied-intelligence/new-relic-alerts/get-started/alerts-best-practices
 ---
 
-This document describes some best practices and examples to help you get the most out of your alerts configuration.
+Improve your Alerts coverage by implementing the following recommendations and get the most out of your alerts configuration. 
 
-<Callout variant="important">
-  Before reading this, we recommend you read [Alerts concepts and workflow](/docs/alerts/new-relic-alerts/getting-started/alert-policy-workflow).
-</Callout>
+Read on to learn the best practices for:
+- Policies
+- Notification channels
+- Incident preferences
+- Thresholds and violations
+- Muting roles
 
-## Define policies for entities or people [#entities-people]
 
-When designing your policies, consider:
+## Recommended alerts [#recommend-alerts]
 
-* The parts of your architecture that need personnel to be responsible for them
-* The individuals who are responsible for one or more parts of your infrastructure
+Use recommended practices if you are new to Alerts or if you want suggestions that optimize your alert coverage. See our [Explorers Hub post on recommended alerts practices](https://discuss.newrelic.com/t/announcing-alert-recommendations-for-apm-applications/154542).
```

Comment:
I feel like it should read "recommended alerts conditions" instead of "recommended practices". There's also an recommended conditions doc here https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations/